File: run_map.r
library(leaflet)
library(htmltools)
library(htmlwidgets)

data <- read.csv2("/Users/Nikon/Desktop/CMC MSU/MC/5 sem/R/maps/coor.txt", sep = "\t", dec = ".", header = TRUE)
map <- leaflet(data) %>% 
  addTiles() %>%
  addMarkers(~data$cor_x, ~data$cor_y, label = ~htmlEscape(data$Num)) %>%
  setView(lng = 2.3, lat = 48.8, zoom = 4.2) %>%
  addProviderTiles("OpenTopoMap")
map
save_html(map, "mapOut.html", background = "white", libdir = "lib")

File: run_map.py
import folium
from folium.plugins import MarkerCluster
import pandas as pd
import html

data = pd.read_table("/Users/Nikon/Desktop/coor.txt", delimiter='\t')
boulder_coords = [48.8, 2.3]
my_map = folium.Map(location = boulder_coords, zoom_start = 4.2, tiles="Stamen Toner")
for i in range(1,21):
	folium.Marker([data['cor_y'][i], data['cor_x'][i]], popup = data['Num'][i]).add_to(my_map)
my_map.save('/Users/Nikon/Desktop/mapOutput.html')
